ts/components/screens/BaseHeader.tsx

```diff
@@ -31,13 +36,13 @@ export class BaseHeader extends React.PureComponent<Props> {
   public render() {
     const { goBack, headerTitle, onShowHelp, body } = this.props;
     return (
-      <AppHeader primary={this.props.primary} noLeft={goBack !== true}>
+      <AppHeader primary={this.props.primary}>
         {goBack && (
           <Left>
             <GoBackButton testID="back-button" onPress={goBack} />
           </Left>
         )}
-        <Body>
+        <Body style={!goBack ? styles.noLeft : {}}>
```

Comment:
invert the logic for clarity:

```suggestion
        <Body style={goBack ? {} : styles.noLeft}>
```